doc/pages/tutorials/010-astarte_in_5_minutes.md

```diff
@@ -70,18 +70,17 @@ $ docker-compose up -d
 
 `docker-compose-initializer` will generate a root CA for devices, a key pair for Housekeeping, and a self-signed certificate for the broker (note: this is a *really* bad idea in production). You can tune the compose file further to use legitimate certificates and custom keys, but this is out of the scope of this tutorial.
 
-Compose might take some time to bring everything up, but usually within a minute from the containers creation Astarte will be ready. Compose will forward the following ports to your machine:
+Compose might take some time to bring everything up, but usually within a minute from the containers creation Astarte will be ready.
+It will bring up a Traefik instance to expose Astarte services. You can reach Astarte at the following addresses:
```

Comment:
I wouldn't mention Traefik here. People Reading this guide are focused on trying out Astarte without going too much into how it  works under the hood